static/js/entry/public.js

```diff
@@ -154,6 +154,8 @@ jQuery(document).ready(function ($) {
     'integrated set of graduate-level online courses. With the MicroMasters ' +
     "credentials, learners can apply for an accelerated master's degree " +
     "program on campus, at MIT or other top universities.";
+  const twitterDescription = 'MITx MicroMasters Programs: a new academic credential ' +
```

Comment:
Can you change the `var description` above to `const description`, so that they both use `const`?